nautobot/ipam/graphql/types.py

```diff
@@ -1,3 +1,5 @@
+import graphene
+
```

Comment:
```suggestion
import graphene
```

Reply:
Sorry not sure what you want me to do here

Reply:
Huh, remove that extra blank line is all. 🙃

Reply:
🤦